src/StackAdmin/Azs.Network.Admin/Module/Azs.Network.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsNetworkQuota.ps1

```diff
@@ -108,7 +108,7 @@ function New-AzsNetworkQuota {
 
     Process {
 
-        $ErrorActionPreference = 'Stop'
+
```

Comment:
Can you please clear this extra whitespace everywhere?

Reply:
Why?  I would have to edit 143+ files to make three blank lines two blank lines.